client/blocks/importer/wordpress/upgrade-plan/index.tsx

```diff
@@ -49,6 +53,17 @@ export const UpgradePlan: React.FunctionComponent< Props > = ( props: Props ) =>
 		isBusy,
 		trackingEventsProps,
 	} = props;
+
+	const importSiteQueryParam = getQueryArg( window.location.href, 'from' )?.toString() || '';
+	const isMigrationModalFeatureEnabled = config.isEnabled( 'migration_assistance_modal' );
```

Comment:
I see that we are already importing `isEnabled`, so there is no need to import config; we can directly use `isEnabled(...`).